README.md

```diff
@@ -115,7 +119,7 @@ On Starknet, the deployment process is in two steps:
    ..
    SN_RPC_URL=<STARKNET_RPC_HTTPS_URL> // Infura or Alchemy RPC URL 
    ETH_CONTRACT_ADDR=0xdd69db25f6d620a7bad3023c5d32761d353d3de9 // GoerliETH 
-   SN_CONTRACT_ADDR=<STARKNET_MM_CONTRACT_ADDR>
+   SN_WALLET_ADDR=<SN_WALLET_ADDR>
    ..
    ```
 2. Declare and Deploy: We sequentially declare and deploy the contracts.
```

Comment:
Please could you add more info about the proxy declaring and deploying?

Reply:
Does this section not get updated in this [PR](https://github.com/yetanotherco/yet-another-bridge/pull/58)? Perhaps, after merging, we can see how it looks and iterate from there.